docs/wearables/providers/heart_rate_zones.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Heart rate zones in workouts"
```

Comment:
Heart Rate Zone? Seems a bit long for a title on the sidebar.

So far Heart Rate Zone is a concept exclusive to workouts in our system. I am not sure if we need to specifically denote "workout" in this case.